src/main/java/Deadline.java

```diff
@@ -0,0 +1,63 @@
+import java.util.Date;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+
+/**
+ * Represents a deadline in the ChatBot task list.
+ * Contains date and time of due deadline and description of the task.
+ */
+public class Deadline extends Task {
+
+    public String dateTime;
+    public LocalDateTime date;
+
+    /**
+     * Constructor for the Deadline task.
```

Comment:
In method header comments, the first sentence should start in the form Returns ..., Sends ..., Adds ... (not Return or Returnning etc.). There should also be an empty line between description and parameter section.